docs/dietpi_docs_todo.md

```diff
@@ -144,7 +165,7 @@ Proposal:
 - Harmonize naming
 - List in alphabetical order
 
-Decision?
+**Decision?**
```

Comment:
This would cause a Markdown-lint error with suggestion to use:
```
#### Decision?
```
😄 we have it excluded anyway, just to remember when writing other pages.